tools/tpm2_dump_capability.c

```diff
@@ -600,7 +600,7 @@ dump_tpm_capability (TPMU_CAPABILITIES    *capabilities,
     case TPM_CAP_COMMANDS:
         dump_command_attr_array (capabilities->command.commandAttributes,
                                  capabilities->command.count);
-        /* fallthrough */
+        break;
```

Comment:
This changes the semantic, now  dump_tpm_capability() no longer returns 1 for TPM_CAP_COMMANDS. Does that mean that gcc7 has found a legitimate bug by ignoring the "/* fallthrough */"?